epub33/core/index.html

```diff
@@ -5984,10 +5984,119 @@ <h4>URLs in the OCF Abstract Container</h4>
 
 					</div>
 
-					<p> In the <a>OCF Abstract Container</a>, when a file uses a URL string to reference another file in
-						the container, the string MUST be a <a data-cite="url#path-relative-scheme-less-url-string"
-							>path-relative-scheme-less-URL string</a>, optionally followed by <code>U+0023 (#)</code>
-						and a <a data-cite="url#url-fragment-string">URL-fragment string</a>. </p>
+					<p>
+						A string is a <dfn id="dfn-valid-relative-container-url-string">valid-relative-container-URL-string</dfn> if it is 
```

Comment:
super-nitpicking-friday-night comment: 
  * if we want to be consistent with the URL standard way of naming things, no last hyphen between "URL" and "string".
  * maybe naming it "relative-container-URL-with-fragment string" would be more symmetric with "absolute-URL-with-fragment string"?

more important: we need to define the  input variable _url_ of the algorithm. s/A string/A string _url_/ should work.

Reply:
yes, yes, and yes :-)